mix.exs

```diff
@@ -91,7 +91,7 @@ defmodule Nostrum.Mixfile do
     [
       {:httpoison, "~> 1.7"},
       {:poison, "~> 3.0"},
-      {:gun, "~> 1.3"},
+      {:gun, github: "ninenines/gun", tag: "2.0.0-rc.1"},
```

Comment:
I'm not sure we can do a hex release with a GH dependency. If we can't we should try and see if one of the other available versions from hex would work. It looks like hex is currently hosting the rc version you have here.

Reply:
Craig, you are absolutely right. I've pinned this to the hex pre-release instead.